common/main/as.xml

```diff
@@ -2294,6 +2294,9 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 			<regionFormat type="daylight">{0} (+1) ডেলাইট সময়</regionFormat>
 			<regionFormat type="standard">{0} (+0) মান সময়</regionFormat>
 			<fallbackFormat>{1} ({0})</fallbackFormat>
+			<zone type="Australia/Currie">
```

Comment:
@macchiati why is `Australia/Currie` moving in all of these?  Kind of seems like a bug, I'd think it should go between `Australia/Broken_Hill` and `Australia/Melbourne`?